zerver/openapi/zulip.yaml

```diff
@@ -12750,6 +12750,57 @@ paths:
                     description: |
                       An example JSON error response for when the public data export
                       exceeds the maximum allowed data export size.
+  /export/realm/consents:
+    get:
+      operationId: get-realm-export-consents
+      summary: Get all private data export consents
+      tags: ["server_and_organizations"]
+      x-requires-administrator: true
+      description: |
+        Fetch the consents of users for their [private data exports](/help/export-your-organization#full-export-with-member-consent)
+        of the organization.
+
+        **Changes**: New in Zulip 10.0 (feature level 294).
```

Comment:
I think this would be clearer:

> Fetches which users have [consented](/help/export-your-organization#full-export-with-member-consent) for their private data to be exported by organization administrators.